node/src/components/linear_chain_sync/state.rs

```diff
@@ -23,8 +24,6 @@ pub enum State {
         /// The most recent block we started to execute. This is updated whenever we start
         /// downloading deploys for the next block to be executed.
         latest_block: Box<Option<Block>>,
-        /// The weights of the validators for latest block being added.
-        validator_weights: BTreeMap<PublicKey, U512>,
```

Comment:
We don't need it b/c we won't be validating finality signatures?

Reply:
We decided that we leave the validation of finality signatures to fast sync, and remove this from the old sync, because it was unused, anyway.